src/containerapp/azext_containerapp/custom.py

```diff
@@ -932,6 +948,34 @@ def update_containerapp_logic(cmd,
     except Exception as e:
         handle_raw_exception(e)
 
+def update_container_app_source(cmd, containerapp_def, name, target_port, image, workload_profile_name, ingress, source, registry_server, registry_user, registry_pass):
+        from ._up_utils import (ContainerApp, ResourceGroup, ContainerAppEnvironment, _reformat_image, _get_registry_details, _has_dockerfile)
+        # Parse resource group name and managed env name
+        env_id = containerapp_def["properties"]['environmentId']
+        parsed_managed_env = parse_resource_id(env_id)
+        env_name = parsed_managed_env['name']
+        env_rg = parsed_managed_env['resource_group']
+
+        # Set image to None if it was previously set to the default image (case where image was not provided by the user) else reformat it
+        image = None if image is None else _reformat_image(source=source,image=image,repo=None)
+        location = containerapp_def["location"]
+        # Construct ContainerApp
+        resource_group = ResourceGroup(cmd, env_rg, location=location)
+        env = ContainerAppEnvironment(cmd, env_name, resource_group, location=location)
+        app = ContainerApp(cmd=cmd, name=name, resource_group=resource_group, image=image, env=env, target_port=target_port, workload_profile_name=workload_profile_name, ingress=ingress, registry_server=registry_server, registry_user=registry_user, registry_pass=registry_pass)
+
+        dockerfile = "Dockerfile"
+        _get_registry_details(cmd, app, source)  # fetch ACR creds from arguments registry arguments
+
+        if source and not _has_dockerfile(source, dockerfile):
+            pass
```

Comment:
This is just a placeholder, correct? So does this if statement serve any purpose?

Reply:
Updated this piece of code in my latest commit.